README.md

```diff
@@ -68,7 +68,7 @@ https://emscripten.org/docs/building_from_source/toolchain_what_is_needed.html.
 - `python`: Version 3.8 or above.
 - `java`: For running closure compiler (optional)
 
-The emsdk pre-compiled binaries are built against Ubuntu/Focal 20.04 LTS and
+The emsdk pre-compiled binaries are built against Ubuntu/Jammy 22.04 LTS and
```

Comment:
Aren't the emsdk binaries built on chromium infra?

Reply:
All the linux machines in the builder bool on the chrome infra are running Jammy already apparently: https://chromium-swarm.appspot.com/botlist?f=cpu%3Ax86&f=host_class%3Adefault&f=os%3AUbuntu-22&f=pool%3Aluci.v8.ci

We might be using a sysroot there to do the actual build, but if we are it will be debian one.

@dschuff do you know if we still use a sysroot ?

Reply:
Split out https://github.com/emscripten-core/emsdk/pull/1633

Reply:
Reverted this

Reply:
But https://github.com/WebAssembly/binaryen/issues/8067 shows an error on the CircleCI bot, not the Chromium infra. And it's using libstdc++ v9.